fairytale.md

```diff
@@ -7,12 +7,6 @@ lives the *Squonk*
 
 The average Squonk is 7 meters tall and screams loud.
 
+The call of a Squonk can be heard miles away, warning you to stay away. (As most try to live a Squonk-free life, the noise is a mild annoyance at best)
 
->  Add some lines to the story. Can be anything. Inspiration?  
```

Comment:
hi @Skelpdar ! Looks like you completely removed these lines but I would like to keep them!  Could you please keep these lines after yours in a way we integrate all of them? Follow slides 84-96 of Tutorial5.
Please complete the changes by Friday 23rh Oct at 18:00, otherwise I will proceed with the changes by some other student.

Let me know if any problem!

Reply:
Those lines are now back again in the branch for this PR. Does this fix the issue?

Reply:
Hi! the lines are back but the history in the two branches diverges. Are you sure you performed a merge? it seems to me that you just added the lines, which is not what is contained in the slides I suggested.

The purpose of a merge is also to reconcile the two histories of different branches into one.

Please follow the directions in slides 84-96 of Tutorial5 regarding how to merge and resolve conflicts.

If the merge is successful, you will not longer see the message below "This branch has conflicts that must be resolved", instead you'll see a message that the merge can be done automatically.

Reply:
You have time to fix until Monday the 26th/Oct at 12. Please let me know if it is not possible.

Reply:
I realized I didn't mention that the real problem is that the master changed in the meantime, see:
https://github.com/floridop/MNXB01-fairytale/commit/0f5ea6f0fac24d41c1a533ecdf28a567d60ca646
There are now new lines before the statement about height, so you need to perform a merge to integrate them. Sorry for the confusion!
Please perform the merge and conflict resolution steps as described in the slides mentioned above.